helmfile.d/helmfile-09.init.yaml

```diff
@@ -45,9 +45,9 @@ releases:
       pkg: tekton-triggers
       app: core
     <<: *default
-  - name: otomi-pipelines
-    installed: true
-    namespace: otomi-pipelines
-    labels:
-      app: core
-    <<: *default
+#  - name: otomi-pipelines
+#    installed: true
```

Comment:
I would actually set it to false, so we ensure that corresponding app is removed